lib/src/test/java/io/ably/lib/types/MessageTest.java

```diff
@@ -46,4 +46,69 @@ public void serialize_message_with_name_and_data() {
         assertEquals("test-data", serializedObject.get("data").getAsString());
         assertEquals("test-name", serializedObject.get("name").getAsString());
     }
+
+    @Test
+    public void serialize_message_with_serial() {
```

Comment:
Do we also need a test (not this file) that handles the inbound stuff - e.g. a quick integration test that ensures fields are set?

Reply:
Yeah, agree to this one 👍

Reply:
@sacOO7 integration test has been added here: [lib/src/test/java/io/ably/lib/test/realtime/RealtimeMessageTest.java](https://github.com/ably/ably-java/pull/1048/files#diff-308e503e1c8c03d7814aa64771fdd96e4d51bb00f0ab44c945cfacf3e396a08f)